wlauto/external/uiauto/src/com/arm/wlauto/uiauto/BaseUiAutomation.java

```diff
@@ -143,12 +143,6 @@ public UiObject getUiObjectByText(String text, String className) throws Exceptio
         return object;
     }
 
```

Comment:
This method name is misleading. I think catching the timeout exception is redundant. It is rarely used. If the timeout exception must be caught I think it is better to do this explicitly.